backend/content/views.py

```diff
@@ -283,8 +315,88 @@ def content_detail(request, content_id):
             "favorite_users": list(content.favorite_users.all().values()),
             "favorite_cnt": content.favorite_cnt,
         }
+
         return JsonResponse(content_detail, safe=False, status=200)
 
+@login_required
+@require_http_methods(["GET"])
+def content_recommendation_2(request, user_id):
+    """
+    /api/content/<int:user_id>/recommendation/
+
+    GET
+        Get recommended contents for current user from THE MOIVE API
+    """
+    if request.method == "GET":
+        try:
+            user = User.objects.get(id=user_id)
+        except User.DoesNotExist:
+            # ERR 404 : User Doesn't Exist
+            return HttpResponse(status=404)
+
```

Comment:
I think we need user configuration, checking if the user_id is the same as the request.user.id